cmd/guacone/cmd/query_vulnerability.go

```diff
@@ -178,15 +178,13 @@ var queryVulnCmd = &cobra.Command{
 						if err != nil {
 							logger.Fatalf("error searching dependency packages match: %v", err)
 						}
-						path = append([]string{certifyVuln.Id}, pkgPath...)
+						path = append([]string{ghsaResponse.Ghsa[0].Id, certifyVuln.Id}, pkgPath...)
 					}
 				}
 			}
 		}
-		//http://localhost:3000/visualize?path=[1,2,3]
 		logger.Infof("found path %v", strings.Join(path, `,`))
 		logger.Infof("Visualizer url: http://localhost:3000/visualize?path=[%v]", strings.Join(path, `,`))
```

Comment:
also need the `PackageName/Namespace/Version` node IDs if you have them (since the UI is filtering based on only what is provided in the path)

Reply:
done